deploy/userdata.sh

```diff
@@ -11,7 +11,7 @@
 apt-get update
 apt-get install ruby -y
 apt-get install wget -y
-cd /home/ubuntu
+cd /home/ubuntu || exit
```

Comment:
This is largely to fix a `shellcheck` warning. We should update the instances with this version at some point, but it's not essential for this to be merged.